tests/rpc/rpc_test.go

```diff
@@ -329,6 +329,26 @@ func waitForReceipt(t *testing.T, hash hexutil.Bytes) map[string]interface{} {
 	}
 }
 
+func TestEth_IncompleteSendTransaction(t *testing.T) {
+	// get gasprice
+	gasPrice := GetGasPrice(t)
+
+	// make tx params without from address
+	param := make([]map[string]string, 1)
+	param[0] = make(map[string]string)
+	param[0]["from"] = ""
+	param[0]["to"] = "0x1122334455667788990011223344556677889900"
+	param[0]["value"] = "0x1"
+	param[0]["gasPrice"] = gasPrice
+	_, err := callWithError("eth_sendTransaction", param)
+
+	// require well-formatted error (should not be "method handler crashed")
+	require.Error(t, err)
+	require.NotEqual(t, err.Error(), "method handler crashed", "no from field dealt with incorrectly")
+	fmt.Println(err.Error())
+	return
```

Comment:
I think we should remove this two lines

Reply:
Also we are trying to migrate our integrations tests to the new python test suite. You could add the test in the `test/integration_tests/test_types.py` file